src/domains/sign/99_main.tf

```diff
@@ -11,23 +11,14 @@ terraform {
   }
 
   backend "azurerm" {}
+
+  required_version = ">= 1.1.7"
 }
 
 provider "azurerm" {
   features {}
 }
 
-data "terraform_remote_state" "core" {
```

Comment:
The state of this domain is still remote on Azure.

`terraform_remote_state` is a separate provider to be used whether a module need some information from *another* state belonging to another module.

In our case, we don't need to do so. I think it was present originally since out terraform files were copied-pasted from another IO domain.